src/api/app/helpers/webui/webui_helper.rb

```diff
@@ -330,9 +330,10 @@ def project_or_package_link(opts)
     defaults = { package: nil, rev: nil, short: false, trim_to: 40 }
     opts = defaults.merge(opts)
 
-    CacheLine.fetch(['project_or_package_link', opts], project: opts[:project], package: opts[:package]) do
-      # only care for database entries
-      prj = Project.where(name: opts[:project]).select(:id, :name).first
+    # only care for database entries
+    prj = Project.where(name: opts[:project]).select(:id, :name).first
+    # Expires in 15 minutes so that changes of local and remote packages eventually result in an update
+    Rails.cache.fetch(['project_or_package_link', prj.cache_key, opts], expires_in: 15.minutes) do
```

Comment:
expires in 15 minutes is kind of 'fuck yourself' - this optimization was mainly for long request lists where we checked for existance and rendered links all the time. 15 minutes is not worth the caching